<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>商品</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_goods_add">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增加商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_goods_delete">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_goods_submit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_goods">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>541</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品信息：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_good">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>330</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_goods_find">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_goods_return">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>会员</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_card_add">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>100</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增加会员</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_card_delete">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>减少会员</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_card_submit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>230</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_card">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>551</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>会员信息：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_card">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>350</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_card_find">
      <property name="geometry">
       <rect>
        <x>692</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_card_return">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>工作人员</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_staff_add">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>80</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增加员工</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_staff_delete">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除员工</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_staff_submit">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_staff">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>会员信息：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_staff">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>330</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_staff_find">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>370</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_staff_return">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>订单</string>
     </attribute>
     <widget class="QTableView" name="tableView_sale">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>290</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>订单信息：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sale">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_find">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>320</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_return">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_add">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增加订单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_delete">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除订单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_submit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>220</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_sale">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>360</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>360</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>订单日期：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sale_find_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>营业额</string>
     </attribute>
     <widget class="QTableView" name="tableView_turnover">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>营业日期：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_turnover_find">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>330</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_turnover_return">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_turnover">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>290</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>供应商</string>
     </attribute>
     <widget class="QTableView" name="tableView_supplier">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>551</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supplier_add">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增加供应商</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supplier_delete">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>200</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除供应商</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supplier_submit">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>280</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>360</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>供应商信息：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supplier">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>360</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supplier_find">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>400</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supplier_return">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_manager_exit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>修改个人信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
